test/test_prompts.py

```diff
@@ -17,6 +17,7 @@ def test_biocypher_prompts(ps):
     ]
     assert list(ps.relationships.keys()) == [
         "Phosphorylation",
+        "GeneToPhenotypeAssociation",
```

Comment:
No, this was correct to not be included. It tests the ability to discard irrelevant associations according to the schema configuration.

Reply:
But, strangely, in my end, the test failed due to "GeneToPhenotypeAssociation" not included.

Reply:
Interesting; we should look into that in the benchmarking module. Not in the tests however, this is LLM behaviour and thus not reliable → should be mocked.

Reply:
you were right, this is a different test than I originally thought. adding the association is correct.